packages/analytics-js-common/src/types/LoadOptions.ts

```diff
@@ -147,6 +147,8 @@ export type LoadOptions = {
   consentManagement?: ConsentManagementOptions;
   sameDomainCookiesOnly?: boolean;
   externalAnonymousIdCookieName?: string;
+  useServerSideCookie?: boolean;
```

Comment:
We don't have to limit ourselves to just a single cookie.
```suggestion
  useServerSideCookies?: boolean;
```

Reply:
Addressed